config.yaml

```diff
@@ -8,6 +8,19 @@ options:
       This feature is experimental and may be unstable.
     type: boolean
     default: False
+  loadbalancer_annotations:
+    description: |
+      A comma-separated list of annotations to apply to the LoadBalancer service.
+      The format should be: `key1=value1,key2=value2,key3=value3`.
+      These annotations are passed directly to the Kubernetes LoadBalancer service, 
+      enabling customization for specific cloud provider settings or integrations.
+      
+      Example:
+        "external-dns.alpha.kubernetes.io/hostname=example.com,service.beta.kubernetes.io/aws-load-balancer-type=nlb"
+
+      Ensure the annotations are correctly formatted and adhere to Kubernetes' syntax and character set : https://kubernetes.io/docs/concepts/overview/working-with-objects/annotations/#syntax-and-character-set
+      invalid values will result in the annotations being reset to None, removed from the LoadBalancer, and all previously set annotations will be lost.
```

Comment:
I worry about the failure case here.  Annotations used on these loadbalancers are sometimes security controls (for example, to make an LB internal-facing only or provide certs).  If someone does this:

```
juju config traefik loadbalancer_annotations="important.security.setting=x,something@malformed"
```

The charm will configure the loadbalancer without the `important.security.setting`.  Yes the charm will complain, but until someone notices and can rectify the issue, the security setting is missing.  

Ideally, I think this should fail in a way that is fail safe, such as removing the LoadBalancer if possible